src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/button.scss

```diff
@@ -0,0 +1,81 @@
+@import '../Application/colors.scss';
+
+$buttonTextColor: $blueZodiac;
+$buttonTextColorActive: $shakespeare;
+$buttonTextColorDisabled: $silver;
+$buttonActiveBackgroundColor: $wildSand;
+
+.button-container {
+    position: relative;
+    display: inline-block;
+}
+
+.button,
+.back-button {
+    display: inline-block;
+    min-width: 60px;
+    position: relative;
+    padding: 0 20px;
+    border: none;
+    height: 60px;
+    background-color: transparent;
+    color: $buttonTextColor;
+    font-size: 12px;
+    cursor: pointer;
+
+    & > * {
+        /* IE11 will emulate 3D effect while clicking button without this */
+        position: relative;
+    }
+
+    &:disabled {
+        color: $buttonTextColorDisabled;
+        cursor: default;
+    }
+
+    &:not(:disabled) {
+        &:hover,
+        &.is-active {
+            color: $buttonTextColorActive;
+            background-color: $buttonActiveBackgroundColor;
+        }
+
+        &.is-active {
+            z-index: 2;
+        }
+    }
+
+    &.is-small {
+        padding: 0;
+
+        .dropdown-icon {
```

Comment:
Any reason this is nested, instead of put into the `.dropdown-icon` class on the outside?